src/main/java/com/halotroop/arrchecker/ARRChecker.java

```diff
@@ -20,11 +20,12 @@ public void onInitialize() {
         for (ModContainer mod : FabricLoader.getInstance().getAllMods()) {
             ModMetadata modMeta = mod.getMetadata();
             // Assume all authors don't know how capitalization works, and don't use an array for the string!
-            String modLicense = modMeta.getLicense().toString().toLowerCase().replace('[', ' ').replace(']', ' ').trim();
+            // '-' is being replaced with ' ' due to some mod authors having specified their creative commons license without '-' but with spaces instead
+            final String modLicense = modMeta.getLicense().toString().toLowerCase().replaceAll("[\\[\\]-]", " ").trim();
             // Create a constant of the mod name and ID to be used multiple times: Don't use an array for the string!
-            String modNameAndID = (modMeta.getName() + " (" + modMeta.getId() + ")").replace('[', ' ').replace(']', ' ').trim();
-            // Give a different warning for Minecraft itself.
-            if (!modMeta.getId().equals("minecraft") && !modMeta.getId().equals("java")) {
+            final String modNameAndID = (modMeta.getName() + " (" + modMeta.getId() + ")").replaceAll("[\\[\\]]", " ").trim();
+            // Ignore built in mods (java, minecraft)
```

Comment:
Minecraft is not a built-in mod. Please don't treat it like one.

Reply:
[It is?](https://github.com/FabricMC/fabric-loader/blob/a0ec81e12aae27e29ee75d14a9248c8bfd7b1098/minecraft/src/main/java/net/fabricmc/loader/impl/game/minecraft/MinecraftGameProvider.java#L80-L96)

Reply:
It is not.